docs/current_docs/manuals/user/cloud/741031-user-interface.mdx

```diff
@@ -3,24 +3,63 @@ slug: /manuals/user/cloud/741031/user-interface
 ---
 
 # User Interface
+:::info
+Dagger 0.11.0 adds support for visualizing Dagger Functions calls in Dagger Cloud. With this update, Dagger Cloud will default to the use the new Traces experience for both `dagger call` and `dagger run` commands. Engines running Dagger 0.11.0 or later will automatically use the new Traces experience. Engines running Dagger 0.10.0 or earlier will continue to use the legacy Runs experience.
+:::
 
-Dagger Cloud lets you visualize your Dagger pipelines in two ways: runs and changes.
-
-## Runs
+Dagger Cloud visualizes your Dagger pipelines runs invoked from your local dev machine and in CI. 
 
 A run is an invocation of a Dagger pipeline. When you run Dagger in CI, each call aligns with one run in Dagger Cloud. If you have many CI jobs configured to run for every pull/merge request, you may see several runs for a single commit.
 
 For instance, if 4 GitHub Actions workflows each call Dagger to run a pipeline for pull requessts, you will see four distinct runs in the Dagger Cloud UI. You may also see multiple runs if one GitHub workflow has multiple jobs defined with distinct calls to Dagger to run your pipelines. As you Daggerize your pipelines, you will likely have fewer distinct CI workflows.
 
 Because Dagger Cloud understands the Dagger engine's data model, it gives you visibility into the steps of your pipeline's Directed Acyclic Graph (DAG), including detailed logs for each step.
```

Comment:
```suggestion
Because Dagger Cloud understands the Dagger Engine's data model, it gives you visibility into the steps of your pipeline's Directed Acyclic Graph (DAG), including detailed logs for each step.
```